lhotse/features/io.py

```diff
@@ -280,7 +280,8 @@ def write(self, key: str, value: np.ndarray) -> str:
         # too many files in a single directory.
         subdir = self.storage_path_ / key[:3]
         subdir.mkdir(exist_ok=True)
-        output_features_path = (subdir / key).with_suffix(".llc")
+        p = subdir / key
+        output_features_path = p.with_suffix(p.suffix + ".llc" if p.suffix != ".llc" else ".llc")
```

Comment:
Is the if/else needed here? (Same for all below changes)

Reply:
The if/else is to prevent adding of suffix if  `p` contains the suffix for some reason say "test.llc". So it will not become "test.llc.llc". 
Which is the expected behavior mentioned here 
https://github.com/lhotse-speech/lhotse/blob/master/lhotse/features/io.py#L443